_posts/2020-02-16-dubdc.md

```diff
@@ -47,9 +45,13 @@ Participants will each be given a 20-minute time slot for both their presentatio
 
 The confirmed panelists are:
 
--TBA
-
-The HUB is accessible by wheelchair. CART and ASL interpretation is available for this event. To request disability accommodation, contact the Disability Services Office at 206-543-6450 (voice), 206-543-6452 (TTY), 206-685-7264 (fax), or dso@uw.edu.
```

Comment:
It appears you accidentally deleted the accessibility data?

Reply:
Ahh thank you so much for catching that. Fixed and updated with the correct building info.